ospfd/ospf_main.c

```diff
@@ -193,15 +191,11 @@ static void ospf_config_end(void)
 /* OSPFd main routine. */
 int main(int argc, char **argv)
 {
-#ifdef SUPPORT_OSPF_API
-	/* OSPF apiserver is disabled by default. */
-	ospf_apiserver_enable = 0;
-#endif /* SUPPORT_OSPF_API */
-
 	frr_preinit(&ospfd_di, argc, argv);
-	frr_opt_add("n:a", longopts,
+	frr_opt_add("n:al:", longopts,
 		    "  -n, --instance     Set the instance id\n"
-		    "  -a, --apiserver    Enable OSPF apiserver\n");
+		    "  -a, --apiserver    Enable OSPF apiserver\n"
+		    "  -l, --apiserver_addr     Set OSPF apiserver bind address\n");
```

Comment:
nit: Can this output be aligned with other args descriptions?

Reply:
Only if I shorten the keyword to something like --api_addr.

Reply:
Note that there is already --vrfdefaultname in the standard FRR daemon paramaters that is not aligned.